packages/agoric-cli/src/deploy.js

```diff
@@ -212,9 +215,9 @@ export default async function deployMain(progname, rawArgs, powers, opts) {
     const wsWebkey = `${wsurl}?accessToken=${encodeURIComponent(accessToken)}`;
 
     const ws = makeWebSocket(wsWebkey, { origin: wsurl.origin });
-    ws.on('open', async () => {
-      connected = true;
-      try {
+    ws.on('open', async () =>
+      (async () => {
```

Comment:
This async IIFE is HUGE, and contain other similar ones. Could we make all big ones into named function expressions, like you did with `promptForUnsafePlugins`? For the big ones, I think the readability benefits of the name outweighs the costs.